src/bots/depositor.py

```diff
@@ -71,6 +70,7 @@ def __init__(
         self._sender = sender
         self._general_strategy = base_deposit_strategy
         self._csm_strategy = csm_strategy
+        self._quorum_cache: Dict[int, datetime] = dict()
```

Comment:
Let's name it `_module_last_heart_beat`. And do:
```
for module_id in DEPOSIT_MODULES_WHITELIST:
    self._module_last_heart_beat[module_id] = now()
```

Reply:
Done